src/index.html

```diff
@@ -20,8 +20,305 @@
+            class="card__button"
+          >
+            BUY
+          </button>
+        </div>
+        <div class="card">
+          <div class="card__img"></div>
+          <a
+            href="#"
+            class="card__name"
+          >
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </a>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="stars">
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon"></div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="price">
+            <div class="price-text">Price:</div>
+            <div class="price__value">$2,199</div>
+          </div>
+
+          <button class="card__button">BUY</button>
+        </div>
+        <div class="card">
+          <div class="card__img"></div>
+          <a
+            href="#"
+            class="card__name"
+          >
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </a>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="stars">
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon"></div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="price">
+            <div class="price-text">Price:</div>
+            <div class="price__value">$2,199</div>
+          </div>
+
+          <button class="card__button">BUY</button>
+        </div>
+        <div class="card">
+          <div class="card__img"></div>
+          <a
+            href="#"
+            class="card__name"
+          >
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </a>
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="stars">
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon stars__icon--active"></div>
+            <div class="stars__icon"></div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="price">
+            <div class="price-text">Price:</div>
+            <div class="price__value">$2,199</div>
+          </div>
+
+          <button class="card__button">BUY</button>
+        </div>
+        <div class="card">
+          <div class="card__img"></div>
+          <a
+            href="#"
```

Comment:
The 'href' attribute is set to '#', which is a placeholder. Consider replacing it with an actual URL to ensure proper navigation.